test/fixtures/responses/index.js

```diff
@@ -1,7 +1,7 @@
-'use strict'; // eslint-disable-line strict
+'use strict' // eslint-disable-line strict
 
 function buildList(options) {
-  return new Array(options.count).fill(options.item);
+  return new Array(options.count).fill(options.item)
```

Comment:
Why are the semicolons removed here?

Reply:
I think this is due to the `prettier`  configuration which indicates that the semicolons are never placed at the end of the rows:

```
{
  "parser": "typescript",
  "printWidth": 80,
  "tabWidth": 2,
  "semi": false,   <== Here
  "singleQuote": true,
  "trailingComma": "none",
  "quoteProps": "consistent",
  "bracketSpacing": false
}
```

Reply:
That seems like a bad configuration. I guess it doesn't matter now anyways, because @ledhed2222 is working on setting up better linting in the repo anyways.